src/api/procedures/issueTokens.ts

```diff
@@ -50,11 +51,15 @@ export async function prepareIssueTokens(
     });
   }
 
-  const defaultPortfolio = await signingIdentity.portfolios.getPortfolio();
+  console.log({ portfolioId: portfolioId?.toString() });
+
+  const portfolio = portfolioId
```

Comment:
I guess we could make this 5.4 compatible by also checking the `isV5` param, but imo its better to wait for us to release the dual version, clean up, and then merge this PR in vs offering an API that won't work as expected on 5.4